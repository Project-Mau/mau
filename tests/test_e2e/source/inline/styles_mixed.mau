You can have _*strong and empashized*_ text.

You can also apply styles to _*`verbatim`*_.

But verbatim will `_*preserve*_` them.

You can use _single *markers.

But you \_need\_ to escape pairs.

Even though you can escape \_only one_ of the two.

If you have \_more than two\_ it's better to just \_escape\_ all of them.

Oh, this is valid for `verbatim as well.
