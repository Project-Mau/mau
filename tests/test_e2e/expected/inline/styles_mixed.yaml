data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'You can have '
      - data:
          args: []
          content:
          - data:
              args: []
              content:
              - data:
                  args: []
                  kwargs: {}
                  subtype: null
                  tags: []
                  type: text
                  value: strong and empashized
              kwargs: {}
              subtype: null
              tags: []
              type: style
              value: star
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: underscore
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' text.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'You can also apply styles to '
      - data:
          args: []
          content:
          - data:
              args: []
              content:
              - data:
                  args: []
                  kwargs: {}
                  subtype: null
                  tags: []
                  type: verbatim
                  value: verbatim
              kwargs: {}
              subtype: null
              tags: []
              type: style
              value: star
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: underscore
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: .
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'But verbatim will '
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: verbatim
          value: _*preserve*_
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' them.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: You can use _single *markers.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: But you _need_ to escape pairs.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: Even though you can escape _only one_ of the two.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: If you have _more than two_ it's better to just _escape_ all of them.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: Oh, this is valid for `verbatim as well.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

